### PR TITLE
fixes tests & status messages

### PR DESCRIPTION
**Pull request by christopherjturner**

### CODE REVIEW

File: src/server/services/about/transformers/service-status.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const githubOrg = config.get('githubOrg')
 
 function serviceStatus(service) {
   const createStatusInfo = service.serviceStatus
-  const createRepository = createStatusInfo?.createRepository
+  const createRepository = createStatusInfo?.['cdp-create-workflows']
```
Comment (feedmypixel) [inline src/server/services/about/transformers/service-status.js:12]: Is this called the right thing now? `createRepository` ?

File: src/server/services/about/transformers/service-status.js [hunk 2]
```diff
@@ -94,9 +96,11 @@ function serviceStatus(service) {
       },
       info: () => {
         switch (cdpAppConfig?.status) {
-          case creationStatuses.raised:
-          case creationStatuses.prOpen:
-            return `Pull request has been raised and will shortly be automatically merged. The GitHub pull request link below has more information.`
+          case creationStatuses.queued:
+            return `Config creation queued.`
```
Comment (feedmypixel) [inline src/server/services/about/transformers/service-status.js:100]: Can be single quotes. Your not doing any interpolation here

File: src/server/test-suites/transformers/test-suite-status.js [hunk 3]
```diff
@@ -42,10 +42,13 @@ function testSuiteStatus(service) {
...
-        text: removeUrlParts(service?.githubUrl),
-        href: service?.githubUrl
-      },
+      url:
+        createRepository?.status === creationStatuses.success
+          ? {
+              text: `${serviceStatus.org}/${serviceStatus.repositoryName}`,
+              href: `https://github.com/${serviceStatus.org}/${serviceStatus.repositoryName}`
+            }
+          : { text: '', href: '' },
```
Comment (feedmypixel) [inline src/server/test-suites/transformers/test-suite-status.js:51]: can this just be null? and guard whatever is consuming it? `url?.text`? `url?.href`?

**Review by feedmypixel — APPROVED** [review]
Some minor comments. LGTM 👍

File: src/server/services/about/transformers/service-status.js (hunk 1 above)
Comment (christopherjturner) [reply src/server/services/about/transformers/service-status.js:12]: `createRepository` is no more, its handled the same as all of the other workflows hence its status field its expecting to receive status updates from.

**Pull request merged**